keps/sig-windows/0000-20181221-windows-group-managed-service-accounts-for-container-identity.md

```diff
@@ -99,105 +101,152 @@ A developer has a Windows web app that they would like to deploy in a container
 
 When this app moves to production on containers, the team will need to coordinate to make sure the right GMSA is carried over to the container deployment.
 
-1. Active Directory domain administrator will:
+1. An Active Directory domain administrator will:
 
   - Join all Windows Kubernetes nodes to the Active Directory domain.
   - Provision the GMSA and gives details to application admin.
   - Assign access to a AD security group to control what machines can use this GMSA. The AD security group should include all authorized Kubernetes nodes.
 
-2. A Kubernetes admin will create a configmap (in the desired namespace) and populate it with configuration information associated with the GMSA:
-  - Run a Windows PowerShell script to verify the account exists, and capture enough data to uniquely identify the GMSA into a JSON file (credspec). This doesn't contain any passwords or crypto secrets.
+2. A Kubernetes cluster admin (or cluster setup tools or an operator) will install a cluster scoped CRD (of kind GMSACredSpec) whose instances will be used to store GMSA credential spec configuration:
```

Comment:
Do we want that in alpha though, instead of using a `ConfigMap`? I thought we wanted to avoid API changes at all for the alpha version?

Reply:
@wk8 using a CRD does not require core API changes. It is a way to introduce/extend Kubernetes.

Reply:
Should have done my research before opening my big mouth, my bad :)

Reply:
Not at all .. good questions - please keep them coming!